README.md

```diff
@@ -61,6 +63,35 @@ in the root of the `meteor` repository. The typical symptom of not
 updating submodules will be `Error: Depending on unknown package ...`
 when you run most Meteor commands.
 
+### Create testing app
```

Comment:
> As far as I can see though it's important that the app gets created by the custom installed version

I'm curious why you feel this way as I don't generally find this to be a problem.  (Though I guess it can be pointed out that an app's `.meteor/release` file can have the value `none` instead of a release number, which might be related to whatever problem you're experiencing).